Intel_SighPP_CMake/Intel_SighPP_CMake/src/priority_lib/priority_module.cpp

```diff
@@ -4,6 +4,35 @@
 #include "priority_module.hpp"
 int pivot = 0;
```

Comment:
I know its not part of this commit, but shouldn't this variable be part of the priority_module?
Instead of being a global variable in this file?

Reply:
Fixed.